code/modules/clothing/suits/shirt.dm

```diff
@@ -1,13 +1,17 @@
-/obj/item/clothing/suit/costume/ianshirt
-	name = "worn shirt"
-	desc = "A worn out, curiously comfortable t-shirt with a picture of Ian. You wouldn't go so far as to say it feels like being hugged when you wear it, but it's pretty close. Good for sleeping in."
-	icon_state = "ianshirt"
-	inhand_icon_state = null
+/obj/item/clothing/suit/costume/wellworn_shirt
+	name = "well-worn shirt"
+	desc = "A worn out, curiously comfortable t-shirt. You wouldn't go so far as to say it feels like being hugged when you wear it, but it's pretty close. Good for sleeping in."
+	icon_state = "wellworn_shirt"
+	inhand_icon_state = null //LEAVE THIS, GAGS INHANDS FUCKING SUUUUCK
```

Comment:
```suggestion
	inhand_icon_state = null
```
No need for the comment lol I only DM'd it with one to explain the thought process